src/ProjectOrigin.Registry/Options/RegistryOptions.cs

```diff
@@ -6,4 +6,6 @@ public record RegistryOptions()
 {
     [Required(AllowEmptyStrings = false)]
     public string RegistryName { get; init; } = string.Empty;
+
+    public bool ReturnComittedForFinalized = false;
```

Comment:
Skal der tilføjes { get; set;} for at den kan (de)serialiseres rigtigt?

Reply:
good catch